examples/counter_functional/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Counter Functional Yew</title>
+  </head>
```

Comment:
Please add `<link data-trunk rel="rust" />` here. See https://github.com/yewstack/yew/commit/daf6ec01523c0ce01f231f17044eace387a46be4 a recent commit of mine for the reason.